### ISSUE TITLE
LUA error when first loading in and when clicking on "Edit note" in context menu.

### ISSUE THREAD

**Issue by NoNick96** [Talryn/CharacterNotes#11]
```
57x Interface/AddOns/CharacterNotes/CharacterNotes.lua:1253: Usage: HookScript(object, method, [handler]): You can only hook a script on a frame object
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0-9/AceHook-3.0.lua"]:143: in function <...lvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:118>
[string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0-9/AceHook-3.0.lua"]:366: in function `HookScript'
[string "@Interface/AddOns/CharacterNotes/CharacterNotes.lua"]:1253: in function <Interface/AddOns/CharacterNotes/CharacterNotes.lua:1249>
[string "=[C]"]: ?
[string "@Interface/AddOns/BagSync/libs/AceAddon-3.0-13/AceAddon-3.0.lua"]:66: in function <...ce/AddOns/BagSync/libs/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@Interface/AddOns/BagSync/libs/AceAddon-3.0-13/AceAddon-3.0.lua"]:523: in function `EnableAddon'
[string "@Interface/AddOns/BagSync/libs/AceAddon-3.0-13/AceAddon-3.0.lua"]:626: in function <...ce/AddOns/BagSync/libs/AceAddon-3.0/AceAddon-3.0.lua:611>
```
Edit note LUA error:

```
2x Interface/AddOns/CharacterNotes/CharacterNotes.lua:1220: attempt to index local 'editwindow' (a nil value)
[string "@Interface/AddOns/CharacterNotes/CharacterNotes.lua"]:1220: in function `EditNoteHandler'
[string "@Interface/AddOns/CharacterNotes/InterfaceMods.lua"]:574: in function `func'
[string "@Interface/AddOns/CharacterNotes/LibDropDownExtension.lua"]:122: in function <...rface/AddOns/CharacterNotes/LibDropDownExtension.lua:97>

Locals:
self = <table> {
 SetDefaultModuleLibraries = <function> defined @Interface/AddOns/BagSync/libs/AceAddon-3.0/AceAddon-3.0.lua:363
 EnableModule = <function> defined @Interface/AddOns/BagSync/libs/AceAddon-3.0/AceAddon-3.0.lua:328
 modules = <table> {
 }
 CancelTimer = <function> defined @Interface/AddOns/BagSync/libs/AceTimer-3.0/AceTimer-3.0.lua:147
 IterateEmbeds = <function> defined @Interface/AddOns/BagSync/libs/AceAddon-3.0/AceAddon-3.0.lua:438
 SecureHookScript = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:409
 GetFontSettings = <function> defined @Interface/AddOns/CharacterNotes/CharacterNotes.lua:130
 RawHookScript = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:394
 CreateNoteLink = <function> defined @Interface/AddOns/CharacterNotes/CharacterNotes.lua:1437
 Printf = <function> defined @Interface/AddOns/BagSync/libs/AceConsole-3.0/AceConsole-3.0.lua:65
 SetDefaultModulePrototype = <function> defined @Interface/AddOns/BagSync/libs/AceAddon-3.0/AceAddon-3.0.lua:405
 optionsFrame = <table> {
 }
 defaultModuleState = true
 UpdateRating = <function> defined @Interface/AddOns/CharacterNotes/CharacterNotes.lua:411
 IsEnabled = <function> defined @Interface/AddOns/BagSync/libs/AceAddon-3.0/AceAddon-3.0.lua:447
 CreateImportPreview = <function> defined @Interface/AddOns/CharacterNotes/Import.lua:177
 ScheduleTimer = <function> defined @Interface/AddOns/BagSync/libs/AceTimer-3.0/AceTimer-3.0.lua:94
 DisableModule = <function> defined @Interface/AddOns/BagSync/libs/AceAddon-3.0/AceAddon-3.0.lua:346
 ShowImportPreview = <function> defined @Interface/AddOns/CharacterNotes/Import.lua:381
 options = <table> {
 }
 db = <table> {
 }
 baseName = "CharacterNotes"
 RegisterMessage = <function> defined @Interface/AddOns/BagSync/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:85
 UnregisterMessage = <function> defined @Interface/AddOns/BagSync/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:148
 RemoveRating = <function> defined @Interface/AddOns/CharacterNotes/CharacterNotes.lua:446
 ImportNotes = <function> defined @Interface/AddOns/CharacterNotes/Import.lua:155
 hooks = <table> {
 }
 ShowNotesImportFrame = <function> defined @Interface/AddOns/CharacterNotes/Import.lua:18
 EasyMenu_Initialize = <function> defined @Interface/AddOns/CharacterNotes/InterfaceMods.lua:197
 LFGListUtil_SetSearchEntryTooltip = <function> defined @Interface/AddOns/CharacterNotes/InterfaceMods.lua:86
 HideNotesWindow = <function> defined @Interface/AddOns/CharacterNotes/CharacterNotes.lua:1338
 EnableInterfaceModifications = <function> defined @Interface/AddOns/CharacterNotes/InterfaceMods.lua:67
 InterfaceModsOptions = <function> defined @Interface/AddOns/CharacterNotes/Options.lua:355
 EditNoteHandler = <function> defined @Interface/AddOns/CharacterNotes/CharacterNotes.lua:1200
 SetEnabledState = <function> defined @Interface/AddOns/BagSync/libs/AceAddon-3.0/AceAddon-3.0.lua:420
 Hook = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:273
 enabledState = true
 NotesExportHandler = <function> defined @Interface/AddOns/CharacterNotes/CharacterNotes.lua:484
 RegisterEvent = <function> defined @Interface/AddOns/BagSync/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:85
 SaveEditNote = <function> defined @Interface/AddOns/CharacterNotes/CharacterNotes.lua:1234
 PARTY_MEMBERS_CHANGED = <function> defined @Interface/AddOns/CharacterNotes/CharacterNotes.lua:1600
 DisableNoteLinks = <function> defined @Interface/AddOns/CharacterNotes/CharacterNotes.lua:1295
 ShowEditNoteFrame = <function> defined @Interface/AddOns/CharacterNotes/CharacterNotes.lua:1153
 IterateModules = <function> defined @Interface/AddOns/BagSync/libs/AceAddon-3.0/AceAddon-3.0.lua:433
 RAID_ROSTER_UPDATE = <function> defined @Interface/AddOns/CharacterNotes/CharacterNotes.lua:1596
 RawHook = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:312
 SetItemRef = <function> defined @Interface/AddOns/CharacterNotes/CharacterNo
```

**Comment by Talryn** [comment]
The patch this week had some big changes.  I believe the version I just released at least fixes this issue.  I need to test everything again when I get more time.

**Comment by NoNick96** [comment]
Fixed in 10.0.6
